lib/lessons.ts

```diff
@@ -1,7 +1,26 @@
 import {request} from 'graphql-request'
 import config from './config'
 
-export async function loadLesson(slug) {
+export async function loadLessons() {
+  const query = /* GraphQL */ `
+    query getLessons {
+      lessons(per_page: 25) {
```

Comment:
This is a placeholder. I'd like to explore the pagination with that new Next.js hook you found.

Reply:
We actively killed the /lessons route in favor of filtered search as a paginated list of 5000+ videos is...

https://twitter.com/chrisbiscardi/status/1172527924945096706?s=20

They capture it in this thread well.